content/en/docs/Examples/services.md

```diff
@@ -462,6 +547,28 @@ service:
       icon: https://raw.githubusercontent.com/hedgedoc/hedgedoc/master/public/icons/android-chrome-512x512.png
 ```
 
+## hoarder-app/hoarder
+Source: https://github.com/hoarder-app/hoarder
+```yaml
+service:
+  Hoarder:
+    options:
+      semantic_versioning: false
+    latest_version:
+      type: github
+      url: hoarder-app/hoarder
+      url_commands:
+        - type: regex
+          regex: v([0-9.]+)$
+    deployed_version:
+      method: GET
+      url: https://hoarder.example.io/
+      regex: \\"serverVersion\\":\\"([0-9.]+)\\"
```

Comment:
```suggestion
      regex: '"serverVersion":"([0-9.]+)"'
```
I think this would be the same and looks a lot simpler (like your Plex one)

Reply:
I just tested this one, it doesn't work with your suggestion... as it's parsing html not json so I need it to be the same as html `\"serverVersion\":\"([0-9.]+)\"`

Reply:
Ah okay, thanks for trying :)